fog/view/enclave/impl/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 #![no_std]
 #![allow(clippy::result_large_err)]
+#![feature(const_cmp)]
```

Comment:
i suspect that this is unnecessary:

if we just did 

```
let LENGTH_TO_COPY: usize = core::cmp::min(FIXED_CIPHERTEXT_LENGTH, ValueSize::USIZE - 1);
```

instead of `const`, it would still be subject to constant folding optimizations by llvm. the only reason to make this `const` here is if you want to use it in a generic parameter to a type or something like this

Reply:
I take the point that it's "one more nightly feature," but I'd prefer to use a seemingly likely-to-stabilize nightly feature and know it's always going to happen vs. depending on the compiler doing a particular thing in a particular way.